File: wasedaorientation2020/f/main.c
#include <stdio.h>

int n, t;
int y[512] = {}, k[512] = {};

int main() {
  scanf("%d", &t);
  while (t--) {
    scanf("%d", &n);
    int win = 0, lose = 0, draw = 0;
    for (int i = 0; i < n; i++) scanf("%d", &y[i]);  //あなたの手の入力
    for (int i = 0; i < n; i++) scanf("%d", &k[i]);  //勝太郎くんの手の入力
    for (int i = 0; i < n; i++) {
      int diff = k[i] - y[i];
      if (diff == 1 || diff == -2)
        ++win;
      else if (diff == 2 || diff == -1)
        ++lose;
      else
        ++draw;
    }
    printf("%d %d %d\n", win, lose, draw);
  }
  return 0;
}

File: wasedaorientation2020/f/Main.java
import java.util.*;

public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int t = sc.nextInt();
        for (int times = 0; times < t; times++) {
            int n = sc.nextInt(), win = 0, lose = 0, draw = 0;
            int[] y = new int[n], k = new int[n];
            for (int i = 0; i < n; i++)
                y[i] = sc.nextInt();
            for (int i = 0; i < n; i++)
                k[i] = sc.nextInt();
            for (int i = 0; i < n; i++) {
                if (y[i] == k[i])
                    ++draw;
                else if ((y[i] - k[i] + 3) % 3 == 2)
                    ++win;
                else
                    ++lose;
            }
            System.out.println(win + " " + lose + " " + draw);
        }
    }
}
